### ISSUE TITLE
Network module, 'Todays estimate' in completed logs.

### ISSUE THREAD

**Issue by hamilton5** [loadavg/loadavg#24]
super minor but, shouldn't transfer 'estimates' become 'totals' when the day ends and the log is complete?

**Comment by loadavg** [comment]
yes - its what they are really :) you are so right... !

**Comment by hamilton5** [comment]
Also bug for network module; after reboots huge huge negitive number logging;
excerpt from log;
1424592901|61.91|28.1
1424593201|92.23|49.88
1424594102|-32803.39|-6555.07
1424594401|0.41|1.91
1424594701|40.29|18.1
1424595001|54.53|24.16

**Comment by loadavg** [comment]
weird - will take a look and see, can you recreate this ?

**Comment by loadavg** [comment]
yes - validated this and uploaded a patch for this to 2.2 branch - though still looking for a fix as to why it actually happens in the logger.
